package/scie-pants.toml

```diff
@@ -40,7 +40,7 @@ members = [
 # N.B.: We name the scie-pants binary scie-pants.bin since the scie itself is named scie-pants
 # which would conflict when packaging.
 name = "scie-pants.bin"
-executable = true
```

Comment:
The error for this one referred to `files[1].executable` which appears to be 1-indexed (AFAICT, this is the first entry in the `files` list); is that expected?

Reply:
It's expected, but perhaps a bad choice. It carried over from my Pex lockfile parsing diagnostics. There I saw a large audience was comprised of ML folks and I've been continually surprised by how much ML they know combined with how little programming language familiarity they have. I should probably just add a parenthetical to diagnostic messages that array indexes are X-based.